docs/faq.md

```diff
@@ -264,7 +264,13 @@ ex.extract("output2", out);
 
 - ## 如何转换 float 数据为 ncnn::Mat
 
-   [use-ncnn-with-pytorch-or-onnx](https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx)
+   首先，自己申请的内存需要自己管理，此时ncnn::Mat不会自动给你释放你传过来的float数据
+   ``` c++
+   std::vector<float> testData(60, 1.0);           // 利用std::vector<float>自己管理内存的申请和释放
+   ncnn::Mat in1(4, 5, 3, (void*)testData.data()); // 把float数据的指针转成void*传过去即可，甚至还可以指定维度
+   float* a = new float[60];         // 自己new一块内存，后续需要自己释放
+   ncnn::Mat in2(4, 5, 3, (void*)a);               // 使用方法和上面相同 
```

Comment:
I suggest constructing with reshape() to handle channel gap issue
```
ncnn::Mat in1 = ncnn::Mat(60, (void*)a).reshape(4, 5, 3);
```

And clone() to transfer data owner
```
ncnn::Mat in1 = ncnn::Mat(60, (void*)a).reshape(4, 5, 3).clone();
```